hammer/synthesis/genus/__init__.py

```diff
@@ -224,7 +224,7 @@ def init_environment(self) -> bool:
         ))
 
         # Load input files and check that they are all Verilog.
-        if not self.check_input_files([".v", ".sv"]):
+        if not self.check_input_files([".v", ".sv", "vh"]):
```

Comment:
This is a duplicate of #823, so I'll close that one.

Reply:
Actually, I think you need to close it. Can you do that and just update the description of this PR?